lib/gringotts/gateways/trexle.ex

```diff
@@ -0,0 +1,92 @@
+defmodule Gringotts.Gateways.Trexle do
+  
+  @moduledoc """
+  Functions supported for working with Trexle payment gateway:
+  *
+  *
+  """
+  @base_url "https://core.trexle.com/api/v1/"
+  @currency "USD"
+  @email "john@trexle.com"
+  @ip_address "66.249.79.118"
+  @description "Store Purchase 1437598192"
```

Comment:
@jyotigautam as discussed with you in person please remove `email`, `ip_address`, `description` and even `currency` these all should be passed from the requests or the config in application in case of currency.